src/XIVLauncher/Game/ProblemCheck.cs

```diff
@@ -89,17 +89,23 @@ public static void RunCheck()
 
             if (d3d11.Exists && dxgi.Exists)
             {
-                if (MessageBox.Show(
-                    Loc.Localize("GShadeError", "A broken GShade installation was detected.\n\nThe game cannot start. Do you want XIVLauncher to fix this? You will need to reinstall GShade."),
-                    "XIVLauncher Error", MessageBoxButton.YesNo, MessageBoxImage.Error) == MessageBoxResult.Yes)
+                var dxgiInfo = FileVersionInfo.GetVersionInfo(dxgi.FullName);
+                var d3d11Info = FileVersionInfo.GetVersionInfo(d3d11.FullName);
+                if (dxgiInfo.ProductName.Equals("GShade", StringComparison.OrdinalIgnoreCase) &&
+                    d3d11Info.ProductName.Equals("GShade", StringComparison.OrdinalIgnoreCase))
                 {
-                    try
-                    {
-                        dxgi.Delete();
-                    }
-                    catch (Exception ex)
+                    if (MessageBox.Show(
+                        Loc.Localize("GShadeError", "A broken GShade installation was detected.\n\nThe game cannot start. Do you want XIVLauncher to fix this? You will need to reinstall GShade."),
+                        "XIVLauncher Error", MessageBoxButton.YesNo, MessageBoxImage.Error) == MessageBoxResult.Yes)
                     {
-                        Log.Error(ex, "Could not delete duplicate GShade.");
+                        try
+                        {
+                            dxgi.Delete();
```

Comment:
We shouldn't try to delete it. Just make an error message. A user could have specific reasons for using a particular injector.

Reply:
No one should use two GShades and I don't want to relocalize this

Reply:
This was pre-existing code and behavior. It doesn't seem to work on Windows at least, but might work on Wine I haven't tested it yet.